backend/src/crd/controller/viewer/reconciler/reconciler.go

```diff
@@ -178,15 +178,23 @@ func setPodSpecForTensorboard(view *viewerV1beta1.Viewer, s *corev1.PodSpec) {
 		fmt.Sprintf("--logdir=%s", view.Spec.TensorboardSpec.LogDir),
 		fmt.Sprintf("--path_prefix=/tensorboard/%s/", view.Name),
 	}
-
-	tfImageVersion := strings.Split(view.Spec.TensorboardSpec.TensorflowImage, ":")[1]
-	// This is needed for tf 2.0
-	if !strings.HasPrefix(tfImageVersion, `1.`) {
+	isTensorflowV1 := false
+	parts := strings.Split(view.Spec.TensorboardSpec.TensorflowImage, ":")
+	// an image might not contain a tag
+	if len(parts) == 2 {
```

Comment:
What is the case if len(parts) !=2, is that invalid? Should we return error if len(parts) !=2 ?

Reply:
e.g. an image without tag is used, it's valid and I think we can now assume it's tensorflow 2+